src/test/java/teammates/test/pageobjects/StudentFeedbackResultsPage.java

```diff
@@ -27,4 +27,20 @@ public String getQuestionAdditionalInfoButtonText(int qnNumber, String additiona
                 By.id("questionAdditionalInfoButton-" + qnNumber + "-" + additionalInfoId));
         return qnAdditionalInfoButton.getText();
     }
+
+    public boolean isExcludingSelfOptionVisible() {
+        return isElementVisible("excluding-self-check");
+    }
+
+    public void clickExcludingSelfCheckbox() {
+        click(By.id("excluding-self-check"));
+    }
+
+    public boolean checkIfExcludingSelfTableShown() {
```

Comment:
The naming of the method is a bit weird. You say `check` while it returns a value. `isExcludingSelfTableVisible()` should be more suitable.